.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+'
+  pull_request: ~
+
+env:
+  COMPOSER_ROOT_VERSION: 1.5.x-dev
```

Comment:
This should not be necessary and `branch-alias` should be used instead:

https://getcomposer.org/doc/articles/troubleshooting.md#dependencies-on-the-root-package

(afaik GH actions do not use shallow clones, or at the very least not shallow enough)

Reply:
The default fetch-depth is 1 (from https://github.com/actions/checkout), Composer was unable to resolve dependencies because of cyclic dependency between AdminUI and ez-support-tools and that's why it was added.

As far as I know Composer should create an implicit branch-alias for branch named like a version (1.5), if I understand correctly your suggestion we could try to add an explicit alias, something like: https://github.com/ezsystems/ezplatform-admin-ui/blob/1.5/composer.json#L46
```
        "branch-alias": {
            "dev-tmp_ci_branch": "1.5.x-dev",
            "dev-master": "1.5.x-dev",
            "dev-1.5": "1.5.x-dev"
        }
```

But I don't know if it works if the whole history is not fetched - definitely something we can check.

Reply:
Ok, didn't know that. If it's necessary for something, then :+1:

Although it makes it rather cumbersome to switch between versions, as you'd need to update both composer.json and CI declaration files. If any solution could be found that is contained within `composer.json` file, I think it should be prefered.